CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 
 ## Unreleased - XXXX-XX-XX
 
+## v0.45.1 - 2021-07-19
+ - Support post-merge and post-commit events
+ - UI - add garbage collection policy configuration
+ - Metastore tool - add flag to support DBFS location
+ - Metastore tool - handle Spark placeholder on copy
+ - Deployment of lakeFS (docker-compose) above MinIO including the following services: Hive metastore, Hive server, Trino and Spark
+
```

Comment:
What about the fixes? I can think of at least 2:
1. Azure block adapter failure to run.
2. S3 404 shenanigans

Reply:
Adding Azure block adapter failure to run
According to S3 404, It wasn't a user facing bug, @itaiad200 do you think we should add it anyway?

Reply:
In that case, no.

Reply:
Also omitted, 

 - Handle dangling commits
 - strip "/" from api base url
 - Align graveler ref explicit branch cases
 - lakectl Show "fs" command, start all command descriptions with capital letter
 
Should I add any of them ?

Reply:
> Also omitted,
> 
> * Handle dangling commits

Yes but with a better description of what was added.

> * strip "/" from api base url

That's in the java/scala client, right? Not really part of this release.

> * Align graveler ref explicit branch cases

I don't know what this is..

> * lakectl Show "fs" command, start all command descriptions with capital letter

Too minor IMO.
 
> Should I add any of them ?

Added my suggestions, which is just opinions so feel free to take it in which direction you prefer.